notebooks/1.data_splits/nbconverted/data_splits.py

```diff
@@ -4,7 +4,7 @@
 # # Spliting Data
 # Here, we utilize the feature-selected profiles generated in the preceding module notebook [here](../0.freature_selection/), focusing on dividing the data into training, testing, and holdout sets for machine learning training.
 
-# In[1]:
+# In[24]:
```

Comment:
Please make sure you run `nbconvert` after fully executing the corresponding jupyter notebooks.